tests/minimal-collectd.conf

```diff
@@ -1,5 +1,12 @@
 Interval 1
 
+LoadPlugin "logfile"
+<Plugin "logfile">
+  LogLevel "debug"
```

Comment:
Current upstream collectd container ships without debug enabled, so this degrades to Info, but I'd still like to keep this config